content/docs/tools-libraries/libraries.md

```diff
@@ -12,6 +12,107 @@ weight: 701
+### Security
+
+This is not a comprehensive security checklist; this only outlines a few
+critical elements. Make sure to run the library through your usual security
+reviews / audits.
+
+-   **Challenge parameters**: Ensure the solution follows the [challenge
+    length](https://www.w3.org/TR/webauthn-3/#sctn-cryptographic-challenges) and
+    [timeout](https://www.w3.org/TR/webauthn-3/#sctn-createCredential)
+    recommended in the specification.
+-   **User ID**: Prioritize solutions that encourage random identifiers, to
+    enhance user privacy. Passkeys should be
+    [free](https://www.w3.org/TR/webauthn-3/#sctn-user-handle-privacy) of
+    personally identifying information (PII). `user.id` should be a [completely
+    random
+    identifier](https://www.w3.org/TR/webauthn-2/#dom-publickeycredentialuserentity-id),
+    and should be different from the ID you've assigned to a user internally.
+-   **Verification steps**: Check whether the library follows verification
+    steps, during
+    [registration](https://developers.google.com/identity/passkeys/developer-guides/server-registration)
+    and
+    [authentication](https://developers.google.com/identity/passkeys/developer-guides/server-authentication).
+
+### UX
+
+If you're looking for a library offering UI elements:
+
+-   **Visual consistency**: Check that the solution uses [standardized
+    icons](https://fidoalliance.org/passkeys/#:~:text=a%20user%E2%80%99s%20passkeys.-,Passkey%20Logo,-Look%20for%20the).
+-   **Clear language**: Instructions using plain language are critical for
+    broader user understanding. Prioritize solutions aligned with the [FIDO UX
+    guidelines](https://fidoalliance.org/ux-guidelines-for-passkey-creation-and-sign-ins/).
+
+More UX/UI guidelines can be found on Google Identity: [Communicating passkeys
+to
+users](https://developers.google.com/identity/passkeys/ux/communicating-passkeys)
+and [Passkeys user interface
+design](https://developers.google.com/identity/passkeys/ux/user-interface-design).
+
+### Developer experience
+
+-   **Full-stack coverage**: A library that offers tightly-integrated frontend
+    and backend components, like in
+    [SimpleWebAuthn](https://simplewebauthn.dev/docs/), can streamline your
+    integration.
+-   **Developer documentation**: Check that the library has a maintained docs website to ease the
+    integration process.
+
+### Developer involvement and maintenance
+
+-   **Standards participation**: Solution maintainers who are active in shaping
+    WebAuthn standards signal expertise. Check whether the maintainers are
+    active in the standards process in some way, in groups such as the [Web
+    Authentication Working Group (WAWG)](https://www.w3.org/groups/wg/webauthn/)
+    or the [WebAuthn Adoption Community Group
+    (WACG)](https://www.w3.org/groups/cg/webauthn-adoption/). The w3c webAuthn
+    [contributor list](https://github.com/w3c/webauthn/graphs/contributors) can
+    be a good place to check.
```

Comment:
Hey @maudnals sorry to restart this conversation this close to getting this guidance in. The rest of the PR is good, but this section in particular might establish some sort of "tier" of third-party OSS libraries. There's nothing inherently "lesser" about a well-maintained library whose maintainer(s) are not active in the WAWG. In fact by this criteria there's probably two, maybe three libraries that would clear most if not all of this guidance and it's simply not factually true that those are the only libraries worth considering.

Not to mention that participation in standards work is unachievable financially and time-wise for many...this is probably too high a bar to set in here.

Reply:
Understood, thank you for the context. I've removed this criteria. 
PTAL at the updated PR, where I've also addressed review comments from @agektmr.